Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/test/TestData.java

```diff
@@ -0,0 +1,20 @@
+package de.rki.coronawarnapp.covidcertificate.test;
+
+public class TestData {
```

Comment:
Could we combine this now or later with `CoronaTestTestData`?

Reply:
This is Java because of the nasty $ signs